app/views/employments/EndEmploymentView.scala.html

```diff
@@ -43,6 +43,7 @@
     pageTitle = Some(Messages("tai.service.navTitle")),
     authedUser = Some(user),
     backLinkId = "backLink",
+    backLinkUrl = Some(controllers.employments.routes.EndEmploymentController.employmentUpdateRemoveDecision.toString),
```

Comment:
you must change the linkId or there's a script that will override this. 
Inspect your links